### PR TITLE
 Add `slime-quicklisp' contrib.

### PR DESCRIPTION
**Pull request by lispnik**
Hi, This adds a new command "quicklisp-quickload" with an alias "ql" for working with quicklisp via the slime repl.

### CODE REVIEW

**Comment by capitaomorte** [comment]
Sent her a thank

**Comment by luismbo** [comment]
@lispnik cool. Care to add a manual entry?

Also, you know what would be uber cool? Something like `M-x list-packages` but for quicklisp dists.

**Comment by lispnik** [comment]
@luismbo Certainly -- manual entry added. I like `M-x list-packages`idea. I'll investigate that.

File: doc/contributors.texi [hunk 1]
```diff
@@ -51,4 +51,5 @@
 @item Brandon Bergren @tab Bozhidar Batsov @tab Bob Halley
 @item Barry Fishman @tab B.Scott Michel @tab Andrew Myers
 @item Aleksandar Bakic @tab Alain Picard @tab Adam Bozanich
+@item Matthew Kennedy
```
Comment (luismbo) [inline doc/contributors.texi]: This file is automatically generated from the ChangeLog. No need to add your name manually.

**Comment by luismbo** [comment]
@lispnik yeah, what @capitaomorte said. Can you conditionalise the swank-loader loading with `#+quicklisp`?

**Comment by lispnik** [comment]
@luismbo @capitaomorte I fixed the quicklisp absence failure but a bit differently to conditionalising with #+quicklisp. In 44cb70e `swank-loader.lisp` still includes `swank-quicklisp` in `*contribs*` but `swank-quicklisp` calls into quicklisp at runtime. If quicklisp is not there, then an error is given. I tested it in these scenarios with CCL and SBCL.

Test 1

1. Quicklisp NOT loaded in Lisp
2. M-x slime
3. ql and quicklisp-quickload commands unavailable

Test 2

1. Quicklisp NOT loaded in Lisp
2. M-x slime
3. ql and quicklisp-quickload commands unavailable
4. M-: (slime-setup '(slime-quicklisp))
5. Switch to REPL, ql and quicklisp-quickload commands ARE available but NOT functional (Error and restart in REPL with "Quicklisp was not already loaded")

Test 3

1. Quicklisp NOT loaded in Lisp
2. M-: (slime-setup '(slime-quicklisp))
3. M-x slime
4. Switch to REPL, ql and quicklisp-quickload commands ARE available but NOT functional (Error and restart in REPL with "Quicklisp was not already loaded")

Test 4

1. Quicklisp IS loaded in Lisp
2. M-: (slime-setup '(slime-quicklisp))
3. M-x slime
4. ql and quicklisp-quickload commands ARE available and functional

**Comment by luismbo** [comment]
@lispnik I'm not sure how often people will load Quicklisp manually, but that seems like a good idea.

**Comment by lispnik** [comment]
@luismbo Sometimes I have a config file like `~/.sbclrc-ql' which I just load when I'm at the REPL. This is mainly when I'm tinkering on Raspberry Pi where the compiler is very slow.

**Comment by luismbo** [comment]
@lispnik I see. Squashed this PR into 59fc3a9 and pushed it to master. Thanks!

**Pull request closed without merging**